bucket.go

```diff
@@ -52,36 +54,54 @@ const (
 
 // BucketStore is a basic interface that describes a bucket - with one or many underlying DataStore.
 type BucketStore interface {
-	GetName() string       // GetName returns the Bucket name
-	UUID() (string, error) // UUID returns a UUID for the bucket
-	Close()                // Close closes the bucket
+	GetName() string       // The bucket's name
+	UUID() (string, error) // The bucket's UUID
+	Close()                // Closes the bucket
 
-	ListDataStores() ([]DataStoreName, error)        // ListDataStores returns a list of all DataStore names in the bucket
-	DefaultDataStore() DataStore                     // DefaultDataStore returns the default data store for the bucket
-	NamedDataStore(DataStoreName) (DataStore, error) // NamedDataStore returns a named data store for the bucket
+	// A list of all DataStore names in the bucket.
+	ListDataStores() ([]DataStoreName, error)
+
+	// The default data store of the bucket (always exists.)
+	DefaultDataStore() DataStore
+
+	// Returns a named data store in the bucket, or an error if it doesn't exist.
+	NamedDataStore(DataStoreName) (DataStore, error)
 
 	MutationFeedStore
 	TypedErrorStore
 	BucketStoreFeatureIsSupported
 }
 
-// DynamicDataStoreBucket is an interface that describes a bucket that can change its set of DataStores.
+// DynamicDataStoreBucket describes a bucket that can create and delete DataStores.
 type DynamicDataStoreBucket interface {
 	CreateDataStore(DataStoreName) error // CreateDataStore creates a new DataStore in the bucket
 	DropDataStore(DataStoreName) error   // DropDataStore drops a DataStore from the bucket
 }
 
 // MutationFeedStore is a store that supports a DCP or TAP streaming mutation feed.
 type MutationFeedStore interface {
+	// The number of vbuckets of this store; usually 1024.
 	GetMaxVbno() (uint16, error)
+
+	// Starts a new DCP event feed. Events will be passed to the callback function.
+	// To close the feed, pass a channel in args.Terminator and close that channel.
+	// - args: Configures what events will be sent.
+	// - callback: The function to be called for each event.
+	// - dbStats: TODO: What does this do? Walrus ignores it.
```

Comment:
Looks like this is solely used as a mechanism to pass through rollback counts as a stat at the moment, and walrus doesn't rollback.

Reply:
Thanks. I've added that to the comments (in both methods where dbStats appears.)